content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,103 @@ description: Upgrading an ambient mode installation with Helm.
 weight: 5
 aliases:
   - /docs/ops/ambient/upgrade/helm-upgrade
-  - /latest/docs/ops/ambient/upgrade/helm-upgrade  
+  - /latest/docs/ops/ambient/upgrade/helm-upgrade
 owner: istio/wg-environments-maintainers
 test: yes
 status: Experimental
 ---
 
 Follow this guide to upgrade and configure an ambient mode installation using
-[Helm](https://helm.sh/docs/). This guide assumes you have already performed an [ambient mode installation with Helm](/docs/ambient/install/helm-installation/) with a previous minor or patch version of Istio.
+[Helm](https://helm.sh/docs/). This guide assumes you have already performed an [ambient mode installation with Helm](/docs/ambient/install/helm-installation/) with a previous version of Istio.
 
 {{< warning >}}
-In contrast to sidecar mode, ambient mode supports moving application pods to an upgraded data plane without a mandatory restart or reschedule of running application pods. However, upgrading the data plane **will** briefly disrupt all workload traffic on the upgraded node, and ambient mode does not currently support canary upgrades of the data plane.
+In contrast to sidecar mode, ambient mode supports moving application pods to an upgraded ztunnel without a mandatory restart or reschedule of running application pods. However, upgrading the ztunnel **will** cause all long-lived TCP connections on the upgraded node to reset, and ambient mode does not currently support canary upgrades of the ztunnel.
 
-Node cordoning and blue/green node pools are recommended to control blast radius of application pod traffic disruption during production upgrades. See your Kubernetes provider documentation for details.
+Node cordoning and blue/green node pools are recommended to limit the blast radius of resets on application traffic during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
+## Understanding Ambient Upgrades
```

Comment:
I'm going to say that I don't know the current capitalization guidelines for ambient, but I still believe they are lower case as is done in other places on this page. I have a few items where I lower case some instances. Feel free to take the advice, or keep what is there.
```suggestion
## Understanding ambient upgrades
```

Reply:
I think in titles we keep it capitalized, but I haven't seen any hard and fast rules... other than it's lowercase in typical use, but follows the language capitalization rules elsewhere (ie, beginning of sentences and titles).

Reply:
"ambient mode" in text, "Ambient mode" at the start of a sentence, whatever works in headings